doc/guide/build_windows.hpp

```diff
@@ -9,8 +9,13 @@
 
 @section build_windows_intro Introduction
 
-This tutorial will show you how to build mlpack for Windows from source, so you can
-later create your own C++ applications. Before you try building mlpack, you may
+This tutorial will how you how to build mlpack for Windows from source, so
```

Comment:
```suggestion
This tutorial will how you show to build mlpack for Windows from source, so
```

Reply:
Thanks for reviewing, I'll fix this now.